Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6-alpine
```

Comment:
> Adds a basic Dockerfile with a base of python:3.7-alpine

I see the 3.6 python.
Anyway I think is good idea to specify full version. For example: `3.6.6-alpine3.8`

Reply:
Good point. Updated in 974f861